### ISSUE TITLE
How is it different from official repo (confluent_kafka)

### ISSUE THREAD

**Issue by sivasai-quartic** [marcosschroh/python-schema-registry-client#58]
@marcosschroh  How is it different from official repo (confluent_kafka) 

I'm able to import MessageSerializer obj from this 

```
from confluent_kafka.avro.serializer.message_serializer import MessageSerializer
```
Any additional features are there?

Thanks

**Comment by marcosschroh** [comment]
Hi @sivasai-quartic 

Yes, you are right. The idea behind of `python-schema-registry-client` was creating a small library that only uses some specific features from `confluent_kafka`, so you do not have to install a big library to use only a few things. 

Some points:

1. We are using the `MessageSerializer` which basically is the same as `confluent_kafka` `MessageSerializer`
2. There is a `Client` that is in charge of managing `avro schemas` which relies on `httpx` and not `requests` as `confluent_kafka` does. 
3. This library includes a `FaustSerializer` that makes an easy integration  with `Faust` framework.
4. This library leaves the door open for `async` support using `httpx`. You can take a look this [project](https://github.com/marcosschroh/async-python-schema-registry-client). When `Faust` has support for `async schemas` I am planning to make this library full `async`
5. The avro schemas are usually hardcoded, and the idea is to integrate the `MessageSerializer` with [dataclasses-avro-schema](https://github.com/marcosschroh/dataclasses-avroschema)

In summary:

If you are using `confluent_kafka` already, does not make sense to use `python-schema-registry-client`. 
If you are using `faust` or you just want to administrate `avro schemas` to interact against the `avro schema server`, this library makes sense. 

I hope this answer your question. Any suggestion is more than welcome.

Thanks
